process/customer_development_playbook.md

```diff
@@ -0,0 +1,321 @@
+# Lab Zero Customer Development Playbook
+
+###### By Clayton Hopkins, Product Designer at [Lab Zero](https://labzero.com/) 
```

Comment:
imo - remove this line in spirit of guides being living docs that team collectively owns.